showthread.php

```diff
@@ -1410,11 +1410,21 @@
 
 		if(is_moderator($forum['fid'], "canopenclosethreads"))
 		{
+			if($thread['closed']) {
```

Comment:
Nitpick: code style:

```suggestion
			if($thread['closed'])
			{
```